win32/build/confutils.js

```diff
@@ -3698,31 +3698,31 @@ function check_binary_tools_sdk()
 function get_clang_lib_dir()
 {
 	var ret = null;
-	var ver = null;
+	var ver = null, major = null;
 
-	if (COMPILER_NAME_LONG.match(/clang version ([\d\.]+) \((.*)\)/)) {
+	if (COMPILER_NAME_LONG.match(/clang version ((\d+)\.[\d\.]+)/)) {
 		ver = RegExp.$1;
+		major = RegExp.$2;
 	} else {
-		ERROR("Failed to determine clang lib path");
+		ERROR("Failed to determine clang version");
 	}
 
-	if (TARGET_ARCH != 'x86') {
-		ret = PROGRAM_FILES + "\\LLVM\\lib\\clang\\" + ver + "\\lib";
+	var cmd = "cmd /c where clang.exe";
```

Comment:
was wondering if llvm-config.exe exists/is well supported on windows but maybe not too useful in those cases.